example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -88,6 +89,10 @@ MM_CollectorLanguageInterfaceImpl::initialize(OMR_VM *omrVM)
 void
 MM_CollectorLanguageInterfaceImpl::flushNonAllocationCaches(MM_EnvironmentBase *env)
 {
+#if defined(OMR_GC_MODRON_STANDARD)
```

Comment:
This should be OMR_GC_MODRON_SCAVENGER as the remembered set is only enabled under that flag.